org.opent2t.sample.hub.superpopular/com.hue.hub/test/mockHueHub.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Potentially, we should visit this in the future and not block this guy.

---
In reply to: [94470370](https://github.com/openT2T/translators/pull/143#discussion_r94470370) [](ancestors = 94470370)